api4jenkins/node.py

```diff
@@ -106,7 +106,7 @@ def __iter__(self):
 
 class MasterComputer(Node):
     def __init__(self, jenkins, url):
-        super().__init__(jenkins, re.sub(r'/master/$', '/(master)/', url))
```

Comment:
You will need both replacements in order to support both old and new versions of Jenkins.
'master' will only turn up on old, and 'Built-In Node' on new versions
Thank you for working on this

Reply:
changed, check again